next-app/src/components/views/MyPage/ InfoRow/InfoRow.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import * as S from './InfoRow.style'
+
+export const InfoRow = ({
+  title,
+  value,
+}: {
+  title: string
+  value: string
+}): JSX.Element => {
```

Comment:
```suggestion
}) => {
```
리턴 타입 명시해주지 않아도 React.ReactNode로 추론해주지 않을까요?
그리고 JSX.Element과 React.ReactNode는 거의 차이가 없지만 개인적으로는 ReactNode 쪽으로 통일해서 쓰는 편입니다

Reply:
네 추론이 가능한 값이라 명시할 필요가 없을 것 같아 삭제 했습니다. JSX.Element보다는 ReactNode가 더 넓은 범주의 값이니 굳이 JSX.Element로 제한해야 하는 경우가 아니면 ReacNode로 사용하는게 좋겠네요